db_upgrade_scripts/mosip_pms/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql

```diff
@@ -402,5 +399,79 @@ UPDATE pms.ftp_chip_detail SET approval_status='approved' where certificate_alia
+VALUES('146211', 'mpartner-default-digitalcard', 'mpolicy-default-PDFCard', 'iris', 'mock', '1.1', 'iris', NULL, 'admin', '2022-02-21 07:02:26.256', 'admin', '2022-02-21 07:02:26.256', false, NULL);
+INSERT INTO pms.partner_policy_bioextract
+(id, part_id, policy_id, attribute_name, extractor_provider, extractor_provider_version, biometric_modality, biometric_sub_types, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+VALUES('146212', 'mpartner-default-digitalcard', 'mpolicy-default-PDFCard', 'fingerprint', 'mock', '1.1', 'finger', NULL, 'admin', '2022-02-21 07:02:26.256', 'admin', '2022-02-21 07:02:26.256', false, NULL);
+
+INSERT INTO pms.partner_policy_credential_type
+(part_id, policy_id, credential_type, is_active, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+VALUES('mpartner-default-digitalcard', 'mpolicy-default-PDFCard', 'PDFCard', true, 'service-account-mosip-regproc-client', '2022-04-04 13:29:10.383', NULL, NULL, false, NULL);
+
+INSERT INTO pms.partner_policy_request
+(id, part_id, policy_id, request_datetimes, request_detail, status_code, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+VALUES('mpartner_policy_PDFCard_req', 'mpartner-default-digitalcard', 'mpolicy-default-PDFCard', '2022-02-21 07:02:26.292', 'mpolicy-default-PDFCard', 'approved', 'admin', '2022-02-21 07:02:26.292', 'admin', '2022-02-21 07:02:26.292', NULL, NULL);
+
+INSERT INTO pms.partner_policy_request
+(id, part_id, policy_id, request_datetimes, request_detail, status_code, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+VALUES('mpartner_policy_digitalcard_req', 'mpartner-default-digitalcard', 'mpolicy-default-digitalcard', '2022-02-21 07:02:26.292', 'mpolicy-default-digitalcard', 'approved', 'admin', '2022-02-21 07:02:26.292', 'admin', '2022-02-21 07:02:26.292', NULL, NULL);
+
+INSERT INTO pms.partner_type (code,partner_description,is_policy_required,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes) VALUES
+	 ('Internal_Partner','Used internally to share certs',true,true,'superadmin',now(),NULL,NULL,false,NULL) ON CONFLICT (code) DO NOTHING;
+
+INSERT INTO pms.partner_type (code,partner_description,is_policy_required,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes) VALUES
+	 ('Print_Partner','Print Partner',true,true,'superadmin',now(),NULL,NULL,false,NULL) ON CONFLICT (code) DO NOTHING;
+
+INSERT INTO pms.partner_type (code,partner_description,is_policy_required,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes) VALUES
+	 ('SDK_Partner','SDK Partner',true,true,'superadmin',now(),NULL,NULL,false,NULL) ON CONFLICT (code) DO NOTHING;
+
+GRANT SELECT, INSERT, TRUNCATE, REFERENCES, UPDATE, DELETE ON ALL TABLES IN SCHEMA pms TO pmsuser;
+
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA pms TO postgres;
```

Comment:
any particular reason why the above 2 statements are moved to the end ? Can't we keep it at the top, it is easier that way since it add it as a convention for every module

Reply:
yes sir, in between we are adding new tables and that tables grant permissions were excluded because those tables were not present when this command was executed. Now that this command is added those tables privileges will also be included.

Reply:
if we want to keep it at the top, I need to add grant statements for those new tables in the script again sir. If that is fine, I'll do that.

Reply:
Yes please do that. Add grants specific to tables

Reply:
okay sir, Updating the same.

Reply:
Updated the same as suggested sir

Reply:
"GRANT ALL PRIVILEGES ON pms.reg_device_type pms TO postgres;" this statement is not required for each table, just granting to the application is good enough.

Reply:
Updated the same as suggested sir